packages/cli/commands/annotate-execution-points-in-repo.ts

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2020-2024 Record Replay Inc. */
+
+import { annotateExecutionData } from "@replay/data/src/analysis/annotateRepoWithExecutionPointData";
+import { AnalysisType } from "@replay/data/src/analysis/dependency-graph-shared";
+import { AnalysisInput } from "@replay/data/src/analysis/dg-specs";
+import {
+  runAnalysisScript,
+} from "@replay/data/src/analysis/run-analysis";
+import { GitRepo } from "@replay/data/src/git-util/git-repos";
+import { fuzzyExtractRecordingAndPoint } from "@replay/data/src/recording-data/point-treasure-hunt";
+import { program } from "commander";
+
+import { printCommandResult } from "../commands-shared/print";
+
+/**
+ * @see https://linear.app/replay/issue/PRO-904/3-let-oh-fix-the-github-issue-using-brians-10609-solution
+ */
+program
+  .command("annotate-execution-points-in-repo")
```

Comment:
<img src="https://github.com/user-attachments/assets/2c20629b-e83f-403b-b233-ebb14e8516ee" alt="A meme with 'Drop the in-repo. It's cleaner.'"/>